### PR TITLE
bash-completion: add --uid and --gid and some trivial change

### PR DESCRIPTION
**Pull request by Kxuan**
add --uid and --gid to bash-completion.

### CODE REVIEW

File: completions/bash/oci-runtime-tool [hunk 1]
```diff
@@ -363,28 +363,36 @@ _oci-runtime-tool_generate() {
...
-		--root-propagation)
-			__oci-runtime-tool_complete_propagations
+		--env-file)
+		    _filedir
+			__oci-runtime-tool_nospace
+		    return
+		    ;;
+
+		--gid)
+            _gids
```
Comment (Mashimiao) [inline completions/bash/oci-runtime-tool]: where is _gids and uids functions?

Comment (Kxuan) [inline completions/bash/oci-runtime-tool]: The _uids and _gids is same as _filedir(can be found at completions/bash/oci-runtime-tool:423). They all are library functions, and be placed in bash-completion.

You can find these functions in /usr/share/bash-completion/bash_completion

**Comment by Mashimiao** [comment]
need reabase

**Comment by Kxuan** [comment]
@Mashimiao PTAL

File: completions/bash/oci-runtime-tool [hunk 2]
```diff
@@ -372,28 +372,36 @@ _oci-runtime-tool_generate() {
...
+		--env-file)
+			_filedir
+			__oci-runtime-tool_nospace
 			return
 			;;
 
-		--cap-add|--cap-drop)
-			__oci-runtime-tool_complete_capabilities
+		--gid)
+            _gids
```
Comment (Mashimiao) [inline completions/bash/oci-runtime-tool]: format with tab

File: completions/bash/oci-runtime-tool [hunk 3]
```diff
@@ -435,14 +442,19 @@ _oci-runtime-tool_generate() {
...
-			_filedir
-			__oci-runtime-tool_nospace
+
+		--seccomp-default)
+			__oci-runtime-tool_complete_seccomp_actions
+			return
+			;;
+
+		--uid)
+            _uids
```
Comment (Mashimiao) [inline completions/bash/oci-runtime-tool]: same here

**Comment by Kxuan** [comment]
@Mashimiao PTAL

**Comment by Mashimiao** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**